rules-reviewed/jboss-eap/websphere/tests/websphere.windup.test.xml

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/v1/xml">
+    <testDataPath>data</testDataPath>
+    <rulePath>../WebSphere.windup.xml</rulePath>
+    <ruleset>
+        <rules>
+            <rule>
```

Comment:
Agreed, I think the test-case rule-id should match the rule-id that is being tested